repo/conf/updates

```diff
@@ -15,3 +15,12 @@ GetInRelease: no
 Architectures: amd64
 VerifyRelease: 74A941BA219EC810
 ListShellHook: grep-dctrl -e -S '^tor(-geoip)?$' || [ $? -eq 1 ]
+
+Name: tor-trixie
```

Comment:
While this is fine to add now, we will also need to add trixie to the list in `.github/workflows/update-tor.yml`.